packages/devtools_app/lib/src/logging/flutter/logging_controller.dart

```diff
@@ -0,0 +1,1000 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
copy of logging/logging_controller.dart to avoid breaking the html app. Changes outlined by comments.